app/assets/stylesheets/custom.scss

```diff
@@ -126,17 +127,24 @@ a {
 
 // profile styling
 
+
+.edit-header {
+  background: #faf8f2;
+  }
+
+.edit-body {
+  background: #faf8f2;
```

Comment:
Color literals like `#faf8f2` should only be used in variable declarations; they should be referred to via variable everywhere else.